src/components/formProducts/FormProducts.js

```diff
@@ -2,12 +2,24 @@ import React, { useRef, useState } from 'react';
 import { Outlet } from 'react-router-dom';
 import { getTokenFromStorage } from '../../utils/utils';
 import { ProductForList } from '../ProductForList';
+import './FormProducts.css';
 
 const FormProducts = ({ items }) => {
   const [itemName, setItemName] = useState('');
   const handleChange = (e) => setItemName(e.target.value);
   const token = useRef(getTokenFromStorage());
 
+  items = !itemName
+    ? items
+    : items.filter((filterItem) =>
+        filterItem.name.toLowerCase().includes(itemName.toLocaleLowerCase()),
+      );
+
+  const resetInput = (e) => {
+    e.preventDefault();
```

Comment:
I don't think you need to prevent the form since the button is not `type='submit'`

Reply:
then I believe it is better to change the button type to 'button', since it is what is actually doing